pkg/utils/common/controller.go

```diff
@@ -149,3 +152,23 @@ func (s *SuiteController) GetRoleBinding(rolebindingName, namespace string) (*rb
 func (s *SuiteController) GetServiceAccount(saName, namespace string) (*corev1.ServiceAccount, error) {
 	return s.KubeInterface().CoreV1().ServiceAccounts(namespace).Get(context.TODO(), saName, metav1.GetOptions{})
 }
+
+func (s *SuiteController) CreateRegistryAuthSecret(secretName, namespace, secretData string) (*corev1.Secret, error){
+	rawDecodedText, err := base64.StdEncoding.DecodeString(secretData)
+    if err != nil {
+        panic(err)
```

Comment:
return an error and don't panic. If panic here the trace should not be human readable